### PR TITLE
Data Upload MongoDB and Raw Data Format Changes

### PR DESCRIPTION
**Pull request by Shkev**
Updating raw data extraction file as well to prevent NaN data values in certain data entries. Note raw data headers changed to not have spaces.

### CODE REVIEW

File: data/mongodb_upload/raw_course_data_upload.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+course_data = pd.read_csv("../course_data_extraction/all_course_data.csv")
+# cleaning some fields
+course_data["Location"] = course_data["Location"].fillna("UNKNOWN")
+course_data["CourseNumber"] = course_data["CourseNumber"].apply(lambda s: s.replace(" ", "-"))
+
+course_data_dict = course_data.to_dict("records")
+
+# Uploading data to database
+print("Inserting data into database...")
+collection.insert_many(course_data_dict)
```
Comment (xuxey) [inline data/mongodb_upload/raw_course_data_upload.py:26]: To remain consistent and maintain one mongodb connection, use Max's Database class from database.py. I believe there is an example usage in course.py.

**Review by xuxey — CHANGES_REQUESTED** [review]
Looks good, Once this implementation uses the Database class (database.py) we can merge this in. Thanks!

File: data/mongodb_upload/raw_course_data_upload.py (hunk 1 above)
Comment (Shkev) [reply data/mongodb_upload/raw_course_data_upload.py:26]: Do we still need this or can the PR be closed?

**Comment by xuxey** [comment]
Since we're switching to SQLite for relational data storage, this is no longer within the project scope.

**Pull request closed without merging**